core/src/toga/widgets/base.py

```diff
@@ -73,7 +73,12 @@ def __repr__(self):
 
     @property
     def id(self) -> str:
-        """The unique identifier for the widget."""
+        """The DOM identifier for the widget.
+
+        This id can be used to target CSS directives.
+
+        :returns: A DOM identifier for the widget.
```

Comment:
Properties don't need a return type; the type annotation of the property is displayed in the documentation.
```suggestion
```